testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oidc/UserInfoTest.java

```diff
@@ -675,4 +703,73 @@ private void testSuccessSignedResponse(Algorithm sigAlg) throws Exception {
 
         return tokenResponse;
     }
+
+    private void includeRolesInUserInfoEndpoint() {
```

Comment:
Is it possible to merge methods includeRolesInUserInfoEndpoint and excludeRolesInUserInfoEndpoint into single method with the boolean parameter? Something like:
```
private void switchIncludeRolesInUserInfoEndpoint(boolean includeRoles)
```
This should avoid some code duplications as both methods are quite similar (with the exception of true/false when switching INCLUDE_IN_USERINFO flag).

Reply:
sure, I changed that.